packages/paymasters/src/WrapContract.ts

```diff
@@ -0,0 +1,34 @@
+import { Contract, providers } from 'ethers'
+import { Eip1193Bridge } from '@ethersproject/experimental'
+import { ExternalProvider } from '@ethersproject/providers'
+import { Signer } from '@ethersproject/abstract-signer'
+
+import { TokenPaymasterConfig, TokenPaymasterProvider } from './TokenPaymasterProvider'
+import { GSNDependencies, WrapBridge } from '@opengsn/provider/dist'
+
+export async function wrapContract (
```

Comment:
No, wait, we cannot have top-level global functions with same names in our packages, this is going to be hell. Please rename to `wrapContractTokenPaymaster` or something like that.